spec/models/milestone_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Milestone, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
oh, we should add some "should belong_to" tests in here for completeness sake.
we should also add to the timeline model tests "should have_many milestones"

Reply:
```suggestion
  describe 'associations' do
    it { should belong_to(:timeline) }
  end
```

Reply:
I just pushed a new version. Let me know if I need to add any additional tests